util/util.go

```diff
@@ -172,23 +173,26 @@ func SetupDefaultLoggingConfig(repoPath string) {
 		MaxBackups: 3,
 		MaxAge:     30, // days
 	}
+	w := zapcore.AddSync(lj)
```

Comment:
running into problems here. trying to configure rotating logs according to https://github.com/uber-go/zap/blob/master/FAQ.md#does-zap-support-log-rotation but it's not clear to me how to integrate that into ipfs/go-log because I don't see how the ability to set log backends is exposed by ipfs/go-log.